book/docs/Open_data_code.ipynb

```diff
@@ -2,21 +2,77 @@
  "cells": [
```

Comment:
<p>we should fix the code to eliminate this warning, I'll need to lookup what it wants instead</p>

<pre class="ql-syntax" spellcheck="false">UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument. 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/scaling_science/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>It seems to be a stackstac package warning. We have the latest version of stackstac here, and I tried this cell with the latest version of Pandas (2.2.2) and still returned the same traceback. Seems like unless we apply some patch fix it might be tricky.</p>

Reply:
that's fine to ignore:
* maybe we should file an upstream ticket on stackstac https://github.com/gjoseph92/stackstac/blob/95fa740cb2d19d89285b198471d92520cf31e10c/stackstac/prepare.py#L410
* Any easy way to suppress the warning so it's not in the notebook/website? cc: @chuckwondo

Reply:
<p>best I could find is </p>

<pre class="ql-syntax" spellcheck="false">import warnings
warnings.filterwarnings('ignore')
</pre><p>is that too ugly to put in the top of the cell ...</p>

Reply:
I prefer the removal of warnings over the ugliness of the import. Committing the changes where appropriate.